packages/electron/webpack.config.js

```diff
@@ -1,5 +1,6 @@
 const { resolve, join } = require("path");
 const { EnvironmentPlugin } = require("webpack");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
```

Comment:
This still wasn't enough to get the app to load, there was error after error, and I just gave up at one point.